src/assets/styles/loader.scss

```
@@ -2,144 +2,160 @@
+		letter-spacing: 0.12em;
+
+		&--number {
+			color: var(--cyan);
+			text-shadow: 0 0 8px var(--cyan-glow);
+		}
+	}
+
+	&__width_animation {
+		animation: width_animation 1s forwards;
+
+		@keyframes width_animation {
```

Stylelint (17.4.0)

[error] Expected keyframe name "width_animation" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)